keps/sig-node/2012-expose-image-id-through-downward-api/kep.yaml

```diff
@@ -0,0 +1,40 @@
+title: Expose status.containerStatuses[0].imageID through Downward API
+kep-number: 2012
+authors:
+  - "@mitar"
+owning-sig: sig-node
+status: provisional
+creation-date: 2020-09-24
+reviewers:
+  - TBD
+approvers:
+  - TBD
+prr-approvers:
+  - TBD
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.20"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.20"
+  beta: "v1.21"
+  stable: "v1.22"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: ImageIDDownwardAPI
+    components:
+      - kube-apiserver
+      - kube-controller-manager
```

Comment:
I am not sure which components this touch, help here would be appreciated.

Reply:
kubelet?